packages/rich-text/src/component/event-listeners/input-and-selection.js

```diff
@@ -114,14 +114,13 @@ export default ( props ) => ( element ) => {
 			return;
 		}
 
-		// Ensure the active element is the rich text element.
-		if ( ownerDocument.activeElement !== element ) {
-			// If it is not, we can stop listening for selection changes. We
-			// resume listening when the element is focused.
-			ownerDocument.removeEventListener(
```

Comment:
To do: it would be good to attach one event handler in the first instance, then reuse it for all instances.